game-core/src/main/java/games/strategy/engine/framework/startup/ui/PlayerType.java

```diff
@@ -97,6 +106,9 @@ public Player newPlayerWithName(final String name) {
   public static String[] playerTypes() {
     return Arrays.stream(values())
         .filter(PlayerType::isVisible)
+        .filter(
+            value ->
+                ClientSetting.showBetaFeatures.getValue().orElse(false) || value != FLOW_FIELD_AI)
```

Comment:
This kind of condition is concise, but takes some work to grok. Can we hack the 'isVisible' to contain this logic? (and make it a bit simpler?)